engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala

```diff
@@ -59,17 +59,5 @@ class WorkflowOutputsSpec extends CromwellTestkitSpec {
         allowOtherOutputs = false
       )
     }
-
-    "Allow explicitly named inputs in the output section" ignore {
-      runWdlAndAssertOutputs(
-        sampleWdl = SampleWdl.ThreeStepWithInputsInTheOutputsSection,
-        eventFilter = EventFilter.info(pattern = s"is in a terminal state: WorkflowSucceededState", occurrences = 1),
-        runtime = "",
-        expectedOutputs = Map(
-          "three_step.cgrep.pattern" -> AnyValueIsFine
-        ),
-        allowOtherOutputs = false
-      )
-    }
```

Comment:
I can sort of imagine a future world where workflows are composable, and you want to just "forward" an input through to the next stage in a pipeline of workflows... But for now it doesn't really serve any purpose